### DISCUSSION TITLE
Predefined port for apps/

### DISCUSSION THREAD

**Discussion by ValentynZup** [vercel/turborepo#1273] [Help] [answered]
How to set up apps/ projects running on same port? 
I have apps/landing/ and apps/app/ 
It start some of app/ or landing/ on 3000 and then trying to start the other -> 3000 in use -> starts on 3001
How to start apps on exact same port?

.env
PORT=3000 or 3001
doesn't help

**Comment by ValentynZup** [comment, marked as answer]
Figured it out: set `next run dev -p 3000`  (or 3001) in apps/app/packages.json and apps/landing/packages.json

**Comment by bobbyflowstate** [comment]
Would it be possible to define ports for all apps in a single config file? For example in turbo.json. We have many apps in our monorepo and would like to be able to easily see what app is running under what port. Would make adding a new app and port easier as well.

**Reply by MwashiHillary** [reply]
Yes. I believe one can pass a configuration file via the environment key in turbo.json.
I am however very uncertain given that I am an absolute beginner in programming.
